src/components/Match/MatchLog.jsx

```diff
@@ -89,6 +118,11 @@ class MatchLog extends React.Component {
         <Checkbox defaultChecked label={checkbox.label} onCheck={(event, isChecked) => this.handleCheck(checkbox.label, isChecked)} />
       </span>))}
       <Table data={generateLog(this.props.match, this.state)} columns={logColumns} />
+      <div>{[...Array(7)].map((_, n) => {
+        const runeString = strings[`rune_${n}`];
+        return <ReactTooltip id={runeString}><span>{runeString}</span></ReactTooltip>;
```

Comment:
See matchColumns.jsx for how we handle this on the match pages.  You can do `Object.keys(strings).filter(key => key.indexOf("rune_") === 0)` to get the list of rune keys/strings.